### DISCUSSION TITLE
ERROR: 'gbk' codec can't decode byte 0xa1 in position 82: illegal multibyte sequence

### DISCUSSION THREAD

**Discussion by HGladiator** [microsoft/nni#3824] [Q&A]
**Environment**:
- NNI version:1.8
- NNI mode (local|remote|pai):local
- Client OS:windows10.1809
- Server OS (for remote mode only):
- Python version:3.7.6
- PyTorch/TensorFlow version:tensorflow. 2.0.0
- Is conda/virtualenv/venv used?:conda
- Is running in Docker?:no

**Log message**:
 - nnictl stdout and stderr: 
ERROR: 'gbk' codec can't decode byte 0xa1 in position 82: illegal multibyte sequence
 
**What issue meet, what's expected?**:
when i run cmd`nnictl create --config config_windows.yml`,then stdout decode error,nii cann`t start. 

**How to reproduce it?**: 
only remove chinese Hanzi Comments from config_windows.yml, nni can normal start

**Additional information**:
eg. maxTrialNum: #可选项: local, remote, pai, kubeflow,   remove "可选项"

**Comment by JunweiSUN** [comment]
@HGladiator this issue may related to the encoding of your cmd/powershell, please change your command line's encoding to UTF-8 and try again.

**Comment by HGladiator** [comment]
my code file is utf-8, use git bash.

**Comment by liuzhe-lz** [comment]
We are removing non-ascii characters from example yaml scripts. This needs to go through the translation process so it may take some time.

**Comment by JunweiSUN** [comment]
> my code file is utf-8, use git bash.

you may want to change your git bash encoding to UTF-8: right click on git bash -> options -> Text -> change `Character set` to `UTF-8` -> Save, then try again.

**Comment by HGladiator** [comment]
> > my code file is utf-8, use git bash.
> 
> you may want to change your git bash encoding to UTF-8: right click on git bash -> options -> Text -> change `Character set` to `UTF-8` -> Save, then try again.

yes. it`s utf-8, the gbk ERROR  reappeared.

**Comment by 5wuuy5** [comment]
I meet this problem also. Have you fixed it?

**Comment by liuzhe-lz** [comment]
> 
> 
> I meet this problem also. Have you fixed it?

The Chinese characters inside example config file are merely comments. You can safely remove them.

**Comment by Jia-py** [comment]
I met this problem today and found no way to solve it.
However, after turning off my computer for a while. It works. Hope this help.
